game-porting-toolkit/files/gameportingtoolkit

```diff
@@ -0,0 +1,12 @@
+#!/bin/zsh
+
+if [ -z "$1" ];  then
+	echo "Usage: $0 <wine-prefix-path> <executable>"
+fi
+
+exe_path="cmd.exe"
+if [ ! -z "$2" ]; then
+	exe_path="$2"
+fi
+
+WINEDLLPATH_PREPEND="@@PREFIX@@/libexec/d3dmetal/wine" MTL_HUD_ENABLED=1 WINEESYNC=1 WINEPREFIX="$1" @@PREFIX@@/libexec/game-porting-toolkit/bin/wine64 "$exe_path" 2>&1 | grep "D3DM"
```

Comment:
Why is the stderr of `wine64` being redirected to stdout and why is that stdout being piped through `grep "D3DM"`? Why do you want to throw away all of wine's other output?